src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -72,6 +72,7 @@ const validationSchema = Yup.object().shape<Venue_v2_AdvancedConfig>({
     then: Yup.string().required("Radio stream is required!"),
   }),
   requiresDateOfBirth: Yup.bool().notRequired(),
+  type: Yup.bool().notRequired(),
```

Comment:
`type` is defined on `Room`, not `Venue`. And it is an optional `enum` of `RoomTypes`, not a `boolean`.

```suggestion
```